docker-compose.yaml

```diff
@@ -62,8 +62,10 @@ services:
       - /etc/flagd/flags.json
     volumes:
       - ./config/flagd/flags.json:/etc/flagd/flags.json
-    expose:
-      - "8013"
+    # expose:
+    #   - "8013"
+    ports:
```

Comment:
Please update the readme to include port `8013`.
https://github.com/open-feature/playground#ports-are-not-available